tests/tools.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from copy import copy
+
+def float_equals(x, y, epsilon=0.0001):
```

Comment:
Our testing platform kinda has this built in: https://stackoverflow.com/a/39623614/5053478

Reply:
Thank you for the insight, it is fixed in f59dfc67ef76fe3e2cfd3a2fec4c739307ef62f1